src/prefect/engine/cloud/flow_runner.py

```diff
@@ -154,7 +251,7 @@ def initialize_run(  # type: ignore
         """
 
         # load id from context
-        flow_run_id = prefect.context.get("flow_run_id")
+        flow_run_id = prefect.context.get("flow_run_id") or context.get("flow_run_id")
```

Comment:
Without this change, all the tests still pass, but there currently isn't _too_ much testing of the `Queued` functionality, so I added the default to be safe as we're populating this value in `CloudFlowRunner.run`.